__mocks__/next/link.tsx

```diff
@@ -0,0 +1,24 @@
+const Link = ({ href, children, ...rest }: any) => {
```

Comment:
this means we don't need to mock next/link in each test jest picks it up automatically